benefit/data/sheets/CaseRoles/CaseRoles-nonprod.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "CaseTypeID": "Benefit",
```

Comment:
Sorry I didnt pick it up the first time but is this needed since you deleted the Creator role ?

Reply:
Removed. Confluence page for CaseRoles LLD implementation if required in future [here](https://tools.hmcts.net/confluence/display/CRef/Role+To+Access+Profiles+-+LLD).